src/libraries/Native/Unix/System.Globalization.Native/CMakeLists.txt

```diff
@@ -59,22 +59,27 @@ set(NATIVEGLOBALIZATION_SOURCES
 
 include_directories(${UTYPES_H})
 
-add_library(System.Globalization.Native
-    SHARED
-    ${NATIVEGLOBALIZATION_SOURCES}
-    ${VERSION_FILE_PATH}
-)
+if (CORECLR_BUILD)
```

Comment:
I would call this something else to future proof this for when Mono is going to need to do the same.

Or just not have this at all, and add `include_directories("../Common")` to every shim unconditionally.

Reply:
I could get away by just including it here unconditionally and leaving it for other shims in the common CMakeLists in the `Unix` root directory.